src/vs/workbench/contrib/extensions/browser/extensionsActions.ts

```diff
@@ -121,6 +121,23 @@ export class PromptExtensionInstallFailureAction extends Action {
 			return;
 		}
 
+		if (ExtensionManagementErrorCode.Signature === (<ExtensionManagementErrorCode>this.error.name)) {
+			await this.dialogService.prompt({
+				type: 'error',
+				message: localize('signature verification failed', "{0} cannot verify the '{1}' extension. Are you sure you want to install it?", this.productService.nameLong, this.extension.displayName || this.extension.identifier.id),
```

Comment:
I read the linked issue and saw this:

> We need to make it easier for users to install if an error happens which might be more common due to Marketplace CDN inconsistencies

If this is due to a marketplace issue it would be better if the message would hint at that. My extension recently got a bad review due to a signature verification failure which was not my fault.

Reply:
Marketplace inconsistencies issue is being tracked here - https://github.com/microsoft/vsmarketplace/issues/619 and expecting that to be fixed which that will unblock the installation.

@isidorn Shall we make the message less aggressive now and make it more stricter going forward? Something like

`VS Code cannot verify the Bla extension. Would you like to install it anyways?

Reply:
@sandy081 Damon will start an email thread on this topic and if we should disable signature verification until CDN issue is fixed. We will decide next week.

@Tyriar thanks for feedback